src/pages/learn/txduration.mdx

```diff
@@ -0,0 +1,115 @@
+# Understanding Interchain Transaction Time
+
+There are several factors affecting the duration of a transaction from when it’s initialized to when it’s finalized. Different blockchains are optimized in different ways. Some blockchains will consistently allow for faster transactions than others. Due to the following factors blockchain transaction time can oftentimes be inconsistent even from the same source to the same destination chain.
+
+1. Finality
+
+   - Axelar’s <i>Cross-Chain Gateway Protocol</i> allows it to work with different blockchains regardless of a given blockchain’s finality rules. However, due to the nature of differing finality on different blockchains, some blockchains might experience longer delays than others when confirming an interchain transfer on the source chain. During this process Axelar validators query their RPC endpoints to see if the transaction is final based on the rules of the chain it is interoperating between. Cross-chain services need to wait for transactions to be finalized on the source chain to prevent the risk of the block being reorganized.
+
+2. Network Congestion
+
+   - Blockchains have a finite amount of resources available to process new transactions and the timing of your transaction can play a role when using the network. During times with high transaction volume the limited amount of miners/ validators that are maintaining the network may become overwhelmed. Increased network congestion may occur due to the following.
+     - Network upgrades: These may cause users to flood the network with transactions in response to the upgrade
+     - Data Intensive Application: Very popular applications on a blockchain may congest the network [as we saw with the Sunflower game on Polygon](https://decrypt.co/89913/nft-farming-game-clogging-up-ethereum-scaling-solution-polygon).
+     - Daily Traffic: Some times of the day are simply known to be more congested than others. Though it is hard to find this data for smaller communities. [There are some patterns that have emerged in Ethereum](https://coinmarketcap.com/alexandria/article/what-is-the-best-time-to-transact-on-ethereum-flipside-crypto)
+
+3. Gas Price
```

Comment:
Ya these two are definitely related. I meant to break them up to sort distinguish between times of extreme congestion such as what happened with the Sunflower Game.
And for number three more just a general rule of thumb so even if the network is not overly congested but your deliberately sending a low gas amount with your tx it will be slower than if you're  sending a high gas amount with your tx.